# san2vuln
python ./run.py Final run-patch 1 --experiment-name usenix_rq2_test --retry-cnt 1 --max-retry-cnt 1 --model gpt-4o --version tot --vuln-ids "CVE-2024-24146"